primer-android/src/org/primer/fragment/hot/OptionHotFragment.java

```diff
@@ -329,10 +329,12 @@ public String getSettingName() {
         public String getOptionName(int index) {
             PrimerjSettings.TransactionFeeMode transactionFeeMode = getModeByIndex(index);
             switch (transactionFeeMode) {
+                /*
```

Comment:
These are best deleted instead of getting commented out. Any deleted code can be traced through git history so no use keeping them in comments.